Source/DataSources/KmlDataSource.js

```diff
@@ -2468,6 +2475,7 @@ define([
      * @param {Object} options An object with the following properties:
      * @param {Camera} options.camera The camera that is used for viewRefreshModes and sending camera properties to network links.
      * @param {Canvas} options.canvas The canvas that is used for sending viewer properties to network links.
+     * @param {Ellipsoid} options.ellipsoid The global ellipsoid used for geographical calculations.  If undefined, WGS84 is used.
```

Comment:
`{Ellipsoid} options.ellipsoid The global ellipsoid used for geographical calculations.  If undefined, WGS84 is used.`

 Should be 

`{Ellipsoid} [options.ellipsoid=Ellipsoid.WGS84] The global ellipsoid used for geographical calculations.`

The `[]` are how you show an optional parameter, and we specify the default there.